django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -157,4 +157,25 @@
     window.showAddAnotherPopup = showRelatedObjectPopup;
     window.dismissAddAnotherPopup = dismissAddRelatedObjectPopup;
 
-})();
+    $(document).ready(function() {
```

Comment:
Since we are already inside a $(function(){}) we can just drop the document.ready no?